packages/qwik/src/core/tests/use-task.spec.tsx

```diff
@@ -810,4 +811,21 @@ describe.each([
     const { vNode } = await render(<Cmp />, { debug });
     expect(vNode).toMatchVDOM(<Component>1 2 3 4 7 8 9</Component>);
   });
+
+  it.only('catch the ', async () => {
+    const Cmp = component$(() => {
+      useTask$(() => {
+        if (isServer) {
+          document.body.innerHTML = '';
+        }
+      });
+
+      return <div>1</div>;
+    });
+    try {
+      await render(<Cmp />, { debug });
+    } catch (e: unknown) {
+      expect((e as Error).message.includes('Code(Q51)')).toBeTruthy;
```

Comment:
I think this needs updating? Also test with a different error, check that it does not add the message but still throws